knpu/using-mink.md

```diff
@@ -21,37 +28,52 @@ little while.
 
 Let's move on to important object #2, and the first that we really care about:
 the session. Add `$session = new Session()` and pass it the `$driver` as
-an argument.
+an argument:
+
+[[[ code('f4233a4ff8') ]]]
 
 Think of the session like a browser tab: anything you can do in a tab, you can do
 in a session. And actually, that isn't very much. You can visit URLs,
 refresh, go backwards, go forwards and that's about it. Let's use it to visit
-a very awesome and absurdly-designed site jurassicpark.wikia.com. After that we'll
-just print out a few things about the page like the status code, and the current URL.
+a very awesome and absurdly-designed site "jurassicpark.wikia.com". After that we'll
+just print out a few things about the page like the status code, and the current URL:
+
+[[[ code('e558213f96') ]]]
 
-To execute this, head over to the terminal and run `php mink.php`. Look at that:
-it's printing out 200 and a slightly different URL than we put in our code.
+To execute this, head over to the terminal and run:
+
+```bash
+php mink.php
+```
+
+Look at that: it's printing out 200 and a slightly different URL than we put in our code.
 That makes sense: when you go to the site in a browser, it redirects to the URL
 we see in our terminal. Mink is emulating a real browser by following redirects.
-But in reality, so far, it's making invisible CURL requests: it's not using a
+But in reality, so far, it's making invisible cURL requests: it's not using a
 real browser.
 
 ## Important Object 3: The Page (DocumentElement)
 
-The third important object is called the page. Grab it by saying
-`$page = $session->getPage()`. I want you to think of this as the JQuery object
-or the DOM. Anything you can do with JQuery - like select elements, click links
-and fill out fields - you can do with the page. Less impressively, it also knows
-the HTML of whatever page we're currently on.
+The third important object is called the page. Grab it by saying `$page = $session->getPage()`:
+
+[[[ code('24eebdad21') ]]]
+
+I want you to think of this as the JQuery object or the DOM. Anything you can do
+with JQuery - like select elements, click links and fill out fields - you can do
+with the page. Less impressively, it also knows the HTML of whatever page we're currently on.
 
 ***TIP
```

Comment:
Hmm, in https://github.com/knpuniversity/blog/blob/master/phpstorm-live-templates.md it works fine.